Update from Laura's comments (#26)
OpenLiberty/guide-microprofile-telemetry-jaeger: README.adoc

== What you'll learn
The use of a microservices architecture can increase the difficulty of understanding how services depend on or affect other services. Consequently, this makes it harder to identify the source of latency or inaccuracies.

One way to increase observability of an application is by emitting traces. https://opentelemetry.io/[OpenTelemetry^] is a set of APIs, SDKs, tooling, and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs. MicroProfile Telemetry adopts OpenTelemetry so your applications can benefit from both manual and automatic traces.

Traces represent requests, which can contain multiple operations, or spans. Each span comprises a name, time-related data, log messages, and metadata that describe what occurred during a transaction. Spans are associated with a context, which identifies the request within which the span occurred. Developers can then follow a single request between services through a potentially complex distributed system. Exporters send the data that MicroProfile Telemetry 1.0 collects to Jaeger so you can visualize and monitor the generated spans.

> For simplicity, you'll configure the provided `system` and `inventory` services to use https://www.jaegertracing.io/[Jaeger^] for distributed tracing with MicroProfile Telemetry. You'll run these services in two separate JVMs made of two server instances to demonstrate tracing in a distributed environment as shown in the diagram below.

Reviewer: ```suggestion
The diagram shows multiple services, which is where distributed tracing is really valuable. However, for simplicity, in this guide you'll configure only the `system` and `inventory` services to use https://www.jaegertracing.io/[Jaeger^] for distributed tracing with MicroProfile Telemetry. You'll run these services in two separate JVMs made of two server instances to demonstrate tracing in a distributed environment as shown in the diagram below.
```

include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[]
----

> Annotate the [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. Then, the OpenTelemetry instrumentation will provide a new span for each method. This will allow you to collect and trace the spans across different services. Additionally, you can annotate the [hotspot=spanAttribute file=0]`host` parameter with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.

Reviewer: ```suggestion
The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. The [hotspot=spanAttribute file=0]`host` parameter is also annotated with the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation with a customized value to indicate that it is part of the trace.
```
This whole bit after the initial instruction (Replace the xx class:) is meant to be describing what doing that action has achieved. I've edited this one to show what I mean - happy to chat more if it's not clear. Also, could you check that the last sentence is accurate - I assumed that where it said "Additionally, you can annotate...." it means that that's been done in the updated class file but it may mean you can optionally go on to do that yourself (which seems less helpful tbh).

> The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. After creating a span, you have the option to include desired parameters with their values in the span by using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation. For example, the [hotspot=spanAttribute file=0]`host` parameter can be assigned [hotspot=spanAttribute file=0]`hostname` as its attribute name using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation, which gives the benefit of tracing the parameters within the [hotspot=addMethod file=0]`add` span.

Reviewer: ```suggestion
The [hotspot=addMethod file=0]`add()` and [hotspot=listMethod file=0]`list()` methods are annotated with the [hotspot=addWithSpan hotspot=listWithSpan file=0]`@WithSpan` annotation. The OpenTelemetry instrumentation provides a new span for each method. You can now collect and trace the spans across different services. After creating a span, you have the option to include desired parameters with their values in the span by using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation. For example, the [hotspot=spanAttribute file=0]`host` parameter is assigned [hotspot=spanAttribute file=0]`hostname` as its attribute name using the [hotspot=spanAttribute file=0]`@SpanAttribute` annotation, which gives the benefit of tracing the parameters within the [hotspot=addMethod file=0]`add` span.
```

The example is what is shown in the guide app code (you're not instructing the reader that this is something they _can_ do if they want; it's already been done for them to show them how it can be done).
